odhargo/README.md

```diff
@@ -1,37 +1,49 @@
-# Argo 
+# ODH Argo
+
+![ODH Argo version](https://img.shields.io/badge/ODH_Argo_version-v2.11.7-yellow.svg) <!-- v2.11.7 -->
+![Upstream version](https://img.shields.io/github/v/release/argoproj/argo?label=Upstream%20release)
+
+ODH Argo component installs Argo that is namespace bound and not cluster wide. There are two pods running after installation
 
-Argo component installs Argo version 2.7 that is namespace bound and not cluster wide. There are two pods running after installation
 1. Argo Server
 2. Argo Controller
 
 This Argo installation uses the "k8sapi" executor to work on Openshift.
 
 ### Folders
-There is only one folder, `base`. This folder includes the installation yaml files to all necessary Argo resources that need to be installed
 
-### Argo Portal
+1. `cluster` folder contains all the cluster wide resources required to be installed by the operator before Argo can be deployed. It contains all `CustomResourceDefinitions` and `ClusterRoles` which aggregates permissions for those CRDs to admin/edit/view project roles.
+
+2. There's only a single `base` folder within the `odhargo` folder. This `base` includes all installation yaml files to all necessary namespaced Argo resources.
 
-This installation creates a route to the Argo portal. To access the portal go to `Routes` and click on the `Argo Portal` route.
+### Argo Server
 
+This installation creates a route to the Argo portal. To access the portal go to `Routes` and click on the `Argo Server` route.
 
 ### Installation
+
 To install Argo add the following to the `kfctl` yaml file.
 
-```
-  - kustomizeConfig:
-      repoRef:
-        name: manifests
-        path: odhargo
-    name: odh-argo
+```yaml
+- kustomizeConfig:
+    repoRef:
+      name: manifests
+      path: odhargo
+  name: odh-argo
 ```
 
 ### Run a Workflow
-To run an example workflow, you can use the portal UI to run the example workflow, however the `serviceAccountName` has to be added to the workflow as shown below. We created this example `serviceAccount` as part of the installation to give basic permissions to a workflow. You can edit this `serviceAccount` and add more pemissions if your workflow requires that.
-```
-apiVersion: argoproj.io/v1alpha1
```

Comment:
I think we should keep these instructions for manual creation of `hello-world` workflow but reference that the argo cli can be used to managed workflows from the command line.